components/STable.vue

```
@@ -9,58 +9,64 @@
+              <q-icon name="link" />
+            </a>
             {{ props.row.Name }}
           </q-td>
           <q-td key="Admin" :props="props">
             <div v-if="props.row.Admin.length > 0">
               <q-chip color="pink" text-color="white" dense>Pokaż grupy</q-chip>
               <q-tooltip content-style="font-size: 12px;" :offset="[0, 0]">
-                <span
-                  v-for="(item, index) in props.row.Admin"
-                  :key="`item-${index}`"
-                >
+                <span v-for="(item, index) in props.row.Admin" :key="`item-${index}`">
                   <pre>{{ item.GroupName }}</pre>
                 </span>
               </q-tooltip>
             </div>
-            <div v-else><span>-</span></div>
+            <div v-else>
+              <span>-</span>
+            </div>
           </q-td>
           <q-td key="Moderator" :props="props">
             <div v-if="props.row.Moderator.length > 0">
               <q-chip color="pink" text-color="white" dense>Pokaż grupy</q-chip>
               <q-tooltip content-style="font-size: 12px;" :offset="[0, 0]">
-                <span
-                  v-for="(item, index) in props.row.Moderator"
-                  :key="`item-${index}`"
-                >
+                <span v-for="(item, index) in props.row.Moderator" :key="`item-${index}`">
                   <pre>{{ item.GroupName }}</pre>
                 </span>
               </q-tooltip>
             </div>
-            <div v-else><span>-</span></div>
+            <div v-else>
+              <span>-</span>
+            </div>
           </q-td>
           <q-td key="GroupCount" :props="props">
-            <span>{{
+            <span>
+              {{
               props.row.GroupCount
-            }}</span>
+              }}
+            </span>
           </q-td>
         </q-tr>
       </template>
     </q-table>
+    <div class="q-pa-lg flex flex-center">
+    <q-pagination
```

Pagination is enabled in q-table by default, simply remove hide-bottom props and there is it 👍

---

Damn i should have read docs abouttable first not paginationXD